tests/test_data/polygon-series-coverage-collection.json

```diff
@@ -0,0 +1,135 @@
+{
+    "type": "CoverageCollection",
+    "coverages": [
+        {
+            "type": "Coverage",
+            "domain": {
+                "type": "Domain",
+                "axes": {
+                    "t": {
+                        "dataType": "time",
+                        "values": [
+                            "2016-01-01T00:00:00Z",
+                            "2016-02-01T00:00:00Z",
+                            "2016-03-01T00:00:00Z"
+                        ]
+                    },
+                    "composite": {
+                        "dataType": "Polygon",
+                        "coordinates": [
+                            "x",
+                            "y"
+                        ],
+                        "values": [
+                            [
+                                [
+                                    [
+                                        -105.67217,
+                                        36.02425
+                                    ],
+                                    [
+                                        -105.88091,
+                                        35.24744
+                                    ],
+                                    [
+                                        -105.01299,
+                                        32.0286
+                                    ],
+                                    [
+                                        -103.54082,
+                                        32.07516
+                                    ],
+                                    [
+                                        -104.06816,
+                                        34.75247
+                                    ],
+                                    [
+                                        -105.67217,
+                                        36.02425
+                                    ]
+                                ]
+                            ]
+                        ]
+                    }
+                }
+            },
+            "ranges": {
+                "Lake/Reservoir Storage End of Month": {
+                    "type": "NdArray",
+                    "dataType": "float",
+                    "axisNames": [
+                        "t"
+                    ],
+                    "shape": [
+                        3
+                    ],
+                    "values": [
+                        1.0,
+                        2.0,
+                        42.0
+                    ]
+                }
+            },
+            "domainType": "PolygonSeries"
```

Comment:
I believe this now part of the wrong object (`Coverage`), and should be part of the `Domain` object.